app/helpers/order_helper.rb

```diff
@@ -4,4 +4,8 @@ module OrderHelper
   def last_payment_method(order)
     OrderPaymentFinder.new(order).last_payment&.payment_method
   end
+
+  def outstanding_balance_label(order)
```

Comment:
@luisramos0 you mentioned extracting this logic to a service but it looked to me like it's more a helper type of thing...?